spleeter_service/Dockerfile

```
@@ -0,0 +1,13 @@
+
+FROM python:3.9-slim
+
+WORKDIR /app
+
+# Install Spleeter and required dependencies
+RUN apt-get update && apt-get install -y ffmpeg && apt-get clean
+RUN pip install --no-cache-dir spleeter
+RUN python3 -m pip install -U demucs
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)